src/express-api/server.ts

```diff
@@ -27,18 +29,20 @@ app.use(bodyParser.urlencoded({ extended: true }));
 
 // IPFS API
 app.get('/v1/ipfs/get/:hash', async (req: express.Request, res: express.Response) => {
-  const data = await getJsonFromIpfs(req.params.hash as string);
-  res.json(data);
+  console.log('Hash', req.params.hash);
+  const data = await ipfs.getContentArray([ req.params.hash as IpfsCid ]);
+  const firstElement = getFirstOrUndefinded(data);
+  res.json(firstElement);

```

Comment:
Implement `getContent()` on `SubsocialIpfsApi` and use here:
```suggestion
  const data = await ipfs.getContent(req.params.hash as IpfsCid)
  res.json(data);
```